app/views/users/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<div>
+  <h1><%= @user.email %></h2><br>
+    <% if @user.id == current_user.id %>
+      <%= button_to "Delete Account", user_path(current_user), method: :delete, data: {confirm: "Are you Sure?"} %>
+    <% end %>
+
+  <h2> Playlists </h2>
+    <% @user.playlists.each do |playlist| %>
+    <%= link_to playlist.title, playlist_path(playlist) %><br>
+  <% end %>
+</div>
+
+<h2>Recent Reviews</h2>
+  <% @user.reviews.order('created_at DESC').each do |review| %>
```

Comment:
Rather than ordering the reviews in the view, it might be better to move this functionality into a method on the model:

```ruby
class User
  def recent_reviews
    reviews.order(created_at: :desc)
  end
end
```

Then in your view you can just call that method:

```HTML+ERB
<% @user.recent_reviews.each do |review| %>
```